aws/cf.go

```diff
@@ -20,12 +21,12 @@ const (
 
 // Stack is a simple wrapper around a CloudFormation Stack.
 type Stack struct {
-	name           string
-	dnsName        string
-	scheme         string
-	targetGroupARN string
-	certificateARN string
-	tags           map[string]string
+	name            string
+	dnsName         string
+	scheme          string
+	targetGroupARN  string
+	CertificateARNs []string
```

Comment:
please do not change visibility here, better add a method, for example: Add(certarns ...string)